src/test/java/socialite/logic/parser/EditCommandParserTest.java

```diff
@@ -68,25 +66,11 @@ public void parse_invalidValue_failure() {
         CommandParserTestUtil.assertParseFailure(
                 parser, "1" + CommandTestUtil.INVALID_PHONE_DESC, Phone.MESSAGE_CONSTRAINTS
         );
-        // invalid email
```

Comment:
We should add some invalid insta/fb/tiktok handle test in the `EditCommandParserTest`, will set up issue for v1.3